src/kubernetes.cc

```diff
@@ -322,8 +324,34 @@ MetadataUpdater::ResourceMetadata KubernetesReader::GetContainerMetadata(
   const std::string k8s_container_name = boost::algorithm::join(
       std::vector<std::string>{kK8sContainerNameResourcePrefix, namespace_name, pod_name, container_name},
       kResourceTypeSeparator);
+
+  std::vector<std::string> local_resource_ids = {
+    k8s_container_pod,
+    k8s_container_name,
+  };
+
+  if (container_status) {
+    std::size_t docker_prefix_end = sizeof(kDockerIdPrefix) - 1;
+
+    const std::string docker_id =
+        container_status->Get<json::String>("containerID");
+    if (docker_id.compare(0, docker_prefix_end, kDockerIdPrefix) != 0) {
+      LOG(ERROR) << "ContainerID "
```

Comment:
ContainerID -> containerID to match the field name?
Or simply "Container ID"

Reply:
Updated.